bin/utils/buildFonts.js

```diff
@@ -52,6 +54,43 @@ async function buildFonts(targetFile) {
 
   const fonts = [...SYSTEM_FONTS, ...googleFonts.items.map(normalizeFont)];
 
+  const tempFile = 'temp.ttf';
```

Comment:
This whole part should be in its own function that's called in `normalizeFont`

Reply:
That's what I did initially, but then "retrieving additional data" didn't suit normalization for me (~SRP).
Happy to change that and reconstruct the .map to fit async/await.